jdaviz/configs/imviz/tests/test_simple_aper_phot.py

```diff
@@ -412,7 +412,7 @@ def test_annulus_background(imviz_helper):
     assert_allclose(phot_plugin.background_value, bg_4gauss_2)
 
     # Edit the annulus and make sure background updates
-    subset_plugin = imviz_helper.plugins["Subset Tools"]._obj
+    subset_plugin = imviz_helper.plugins["Subsets"]._obj

```

Comment:
Same as above.

Reply:
I'm leaving this as is; otherwise, I'll need to ._obj for several non-exposed APIs. This makes the code cleaner if that’s alright.